runner/src/server/plugins/engine/models/submission/index.ts

```diff
@@ -0,0 +1,3 @@
+export { EmailModel } from "./EmailModel";
```

Comment:
not a huge fan of using the word model at the end of models when you're in a folder called models. Guess thats a standard we've set?

Reply:
No, me neither. It's not a standard we have explicitly set, I would say it's inherited more than anything. FormModel and SummaryViewModel (although technically this is meant to be a VM..) are named that way.

Reply:
I'm happy if you wanna keep it obviously, I just thought I'd raise it and you can do whatever you want with it. Up to you 👀

Reply:
yes, but not for this.. issue opened